client/app.jsx

```diff
@@ -80,10 +87,10 @@ class Location extends React.Component {
     return(
       <div>
         <h2>Location</h2>
-        {(this.state.lat !== null && this.state.lng !== null) ?  <Map id={'map'} style={{zIndex: -1, position: 'relative', width: '100%', height: 300, borderRadius: '12px'}} lat={this.state.lat} lng={this.state.lng}/> : <div>loading...</div>}
+        {(this.state.lat !== null && this.state.lng !== null) ?  <Map id={'map'} style={{zIndex: -1, position: 'relative', width: '100%', height: 300}} lat={this.state.lat} lng={this.state.lng}/> : <div>loading...</div>}
```

Comment:
I see the zIndexing here haha

Reply:
yeah, if I don't do it then Andy's modal is going to be behind my map...